.github/workflows/test.yml

```diff
@@ -16,19 +16,38 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby:
-          - '2.5'
-          - '2.6'
-          - '2.7'
-          - '3.0'
```

Comment:
FYI, using a string for `'3.0'` was a deliberate choice. Changing that to `3.0` changed what's tested here.

![Screenshot from 2022-12-31 13-10-09](https://user-images.githubusercontent.com/3138447/210155663-e9d33ede-8ff9-4552-8db9-f60749e30c4f.png)

GitHub Actions are powered by Node.js, and it can't distinguish `3` and `3.0`.

Reply:
Oh, I wasn't aware of these subtleties.